collect_app/src/main/res/drawable/start_new_form_button_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@color/color_on_surface_low_emphasis">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:radius="100dp" />
+            <solid android:color="?colorAccent" />
```

Comment:
`?colorAccent` is an AppCompat color attribute. We should use `?colorPrimary` here instead.

Reply:
Done.